.github/workflows/production.yml

```diff
@@ -19,64 +19,78 @@ jobs:
         with:
           ref: release
 
-      - uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b # v4
-        with:
-          node-version: "^20"
-          cache: yarn
-          cache-dependency-path: yarn.lock
+      - name: Set VERSION
+        run: echo "VERSION=$(./scripts/get_version.sh)" >> $GITHUB_ENV
 
-      - name: Setup environment
-        run: sudo apt-get install libelf1
+      - name: Write commit SHA to file
+        run: echo $GITHUB_SHA > frontends/main/public/hash.txt
 
-      - name: Install frontend dependencies
-        run: yarn install --immutable
+      - name: Heroku login
+        env:
+          HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+        run: heroku container:login
 
-      - name: Set VERSION
-        run: echo "VERSION=$(./scripts/get_version.sh)" >> $GITHUB_ENV
+      - name: Release Backend on Heroku
+        uses: akhileshns/heroku-deploy@581dd286c962b6972d427fcf8980f60755c15520
+        with:
+          heroku_api_key: ${{ secrets.HEROKU_API_KEY }}
+          heroku_app_name: mitopen-production
+          heroku_email: ${{ secrets.HEROKU_EMAIL }}
+          branch: release
 
-      - name: Build frontend
-        run: NODE_ENV=production yarn build
+      - name: Build and push the Docker image
         env:
+          HEROKU_APP_NAME: mitopen-production-nextjs
+          HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+          ORIGIN: https://learn.mit.edu
+          MITOL_API_BASE_URL: https://api.learn.mit.edu
+          SITE_NAME: MIT Learn
+          SUPPORT_EMAIL: mitlearn-support@mit.edu
           EMBEDLY_KEY: ${{ secrets.EMBEDLY_KEY_PROD }}
-          POSTHOG_ENABLED: true
+          MITOL_AXIOS_WITH_CREDENTIALS: true
+          CSRF_COOKIE_NAME: ${{ secrets.CSRF_COOKIE_NAME_PROD }}
           POSTHOG_API_HOST: https://app.posthog.com
-          POSTHOG_TIMEOUT_MS: 1000
           POSTHOG_PROJECT_ID: ${{ secrets.POSTHOG_PROJECT_ID_PROD }}
-          POSTHOG_PROJECT_API_KEY: ${{ secrets.POSTHOG_PROJECT_API_KEY_PROD }}
+          POSTHOG_API_KEY: ${{ secrets.POSTHOG_PROJECT_API_KEY_PROD }}
```

Comment:
there is a lingering reference to the old variable name in pytest.ini which we might want to update or remove

Reply:
@jonkafton we do now have
```
# rc
SENTRY_PROFILES_SAMPLE_RATE: ${{ secrets.SENTRY_PROFILES_SAMPLE_RATE_RC }}
SENTRY_TRACES_SAMPLE_RATE: ${{ secrets.SENTRY_TRACES_SAMPLE_RATE_RC }}
# and  prod 
SENTRY_PROFILES_SAMPLE_RATE: ${{ secrets.SENTRY_PROFILES_SAMPLE_RATE_PROD }}
SENTRY_TRACES_SAMPLE_RATE: ${{ secrets.SENTRY_TRACES_SAMPLE_RATE_PROD }}
```
available